packages/perseus/src/widgets/graded-group-set.jsx

```diff
@@ -33,35 +33,41 @@ type IndicatorsProps = {|
     onChangeCurrentGroup: (groupNumber: number) => void,
 |};
 
-// TODO(jeremy): This indicator panel is not accessible for keyboard
-// navigation. Add in keyboard handling and tab indexes to make accessible.
 class Indicators extends React.Component<IndicatorsProps> {
-    render(): React.Node {
-        const items = [];
-        for (let i = 0; i < this.props.numGroups; i++) {
-            items.push(
-                <div
-                    role="button"
-                    aria-label={i18n._("Skip to %(title)s", {
-                        title: this.props.groupTitles[i],
-                    })}
-                    key={i}
-                    className={css(
-                        styles.indicator,
-                        i === this.props.currentGroup &&
-                            styles.selectedIndicator,
-                    )}
-                    onClick={() => this.props.onChangeCurrentGroup(i)}
-                >
-                    {i === this.props.currentGroup && (
-                        <span className={css(a11y.srOnly)}>
-                            {i18n._("Current")}
-                        </span>
-                    )}
-                </div>,
-            );
+    handleKeyDown = (e: SyntheticKeyboardEvent<>, i: number) => {
+        if (e.key === "Enter") {
```

Comment:
Just a thought: with keyboard navigation, I think it's more common to use "Space" vs "Enter" when selecting ("Enter" usually means submit; for instance with native radios in forms). I guess this is kind of a submit action though, like a pagination request. idk, just thinking out loud. 😅

Reply:
+1

Reply:
@Myranae Thanks for the link! I'll make sure enter and space are covered.